lexicons/app/bsky/unspecced/registerPushNotificationEndpoint.json

```diff
@@ -0,0 +1,20 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.unspecced.registerPushNotificationEndpoint",
```

Comment:
I'm not sure the different between these routes, but I like this name better - altho I would just do `registerPushNotifications`

Reply:
Changed it to `registerPushNotifications`